sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -923,10 +931,13 @@ private void postRegister() {
         revive();
     }
 
-    @Override
-    public void update(Observable observable) {
-        if (observable == planCoordinator) {
-            suppressOrRevive();
-        }
+    private Plan getDeployPlan() {
+        return getDeployPlan(plans).get();
```

Comment:
So I don't have to duplicate code for finding the plan named `deploy` in a list of plans.